apps/qa/src/README.md

```diff
@@ -12,31 +12,128 @@ npm install
 
 ## Configuration
 
-The app requires you to configure a `.env` file at the root of the project with, at minimum, these properties:
+The app requires you to configure a `configuration.ts` file in the `/src` directory, with these properties, at minimum:
 
-```yaml
-REACT_APP_corpus_id=<your corpus ID here>
-REACT_APP_customer_id=<your customer ID here>
-REACT_APP_api_key=<your API key here>
-REACT_APP_endpoint=api.vectara.io
+```
```

Comment:
We can add the `ts` annotation here and to the other code blocks to add syntax-highlighting.

Also can we update this code example to demonstrate importing the `Config` type?

```ts
import { Config } from "./contexts/ConfigurationContext";

export const configuration: Config = {
  corpusId: "<your corpus ID here>",
  customerId: "<your customer ID here>",
  apiKey: "<your API key here>",
  endpoint: "api.vectara.io"
};
```